Rename module `xclim.indices` to `xclim.compute` (#2330)
Ouranosinc/xclim: docs/explanation.rst

Purpose
=======

`xclim` aims to position itself as a climate services tool for any researchers interested in using Climate and Forecast Conventions (`CF-Conventions <https://cfconventions.org/>`_) compliant datasets to perform climate analyses.
This tool is optimized for working with Big Data in the climate science domain and can function as an independent library for one-off analyses in *Jupyter Notebooks* or as a backend engine for performing climate data analyses via **Web Processing Services** (`WPS <https://www.ogc.org/standard/wps/>`_; e.g. `Finch <https://github.com/bird-house/finch>`_).
It was primarily developed targeting Earth and Environmental Science audiences and researchers, originally for calculating climate indicators for the Canadian government web service `ClimateData.ca <https://climatedata.ca/>`_.

> The purposes that `xclim` is built for are in calculating climate indicators, evaluating the sanity of outputs of climate models, and in performing climate model simulation ensemble statistics.

Reviewer: ```suggestion
`xclim` is built for many purposes: calculating climate indicators, evaluating the sanity of outputs of climate models, and performing climate model simulation ensemble statistics.
```

The sentence seems awkward, I suggest this, or something else
